### PR TITLE
Display real json data from xdr

### PR DESCRIPTION
**Pull request by 0xExp-po**

### CODE REVIEW

File: AES256-encryptor/.gitignore
Comment (tupui) [inline AES256-encryptor/.gitignore:1]: I think we can delete the whole folder now.

File: dapp/src/utils/stellarXdr.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+declare global {
+  interface Window {
+    __STELLAR_XDR_INIT__?: boolean;
+  }
+}
+
+const initialize = async () => {
+  if (!window.__STELLAR_XDR_INIT__) {
+    await init(wasmUrl);
+    console.log("Stellar XDR JSON initialized");
```
Comment (tupui) [inline dapp/src/utils/stellarXdr.ts:19]: I think we can cleanup all log

File: dapp/src/utils/stellarXdr.ts
Comment (tupui) [inline dapp/src/utils/stellarXdr.ts:1]: Why is this needed? Please add a comment

**Review by tupui — CHANGES_REQUESTED** [review]
Looks good, just some cleanup I believe

File: dapp/src/utils/stellarXdr.ts
Comment (0xExp-po) [reply dapp/src/utils/stellarXdr.ts:1]: stellar-xdr-json module is the best module to convert xdr to json data.
but this module is built with rust and we need initialize the module wasm to use this module.
So this initialize function is required and I added this

**Review by tupui — APPROVED** [review]

**Pull request merged**